pkg/porter/list_test.go

```diff
@@ -4,19 +4,19 @@ import (
 	"context"
 	"testing"
 
-	"get.porter.sh/porter/pkg/claims"
 	"get.porter.sh/porter/pkg/cnab"
 	"get.porter.sh/porter/pkg/secrets"
+	"get.porter.sh/porter/pkg/storage"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
 func TestNewDisplayInstallation(t *testing.T) {
 	t.Run("installation has been installed", func(t *testing.T) {
-		cp := claims.NewTestClaimProvider(t)
+		cp := storage.NewTestClaimProvider(t)
```

Comment:
Do we want to also change the `NewTestClaimProvider` to `NewTestStorageProvider` ?

Reply:
That is in the follow-up PR #2053